cosmos/config.py

```diff
@@ -236,6 +240,44 @@ def validate_profile(self) -> None:
                 "Both profiles_yml_filepath and profile_mapping are defined and are mutually exclusive. Ensure only one of these is defined."
             )
 
+    def validate_dbt_config_vars(self) -> None:
+        "Validates config vars for profile."
+
+        vars_checks = {
+            "send_anonymous_usage_stats": {'var_type': bool},
+            "use_colors": {'var_type': bool},
+            "partial_parse": {'var_type': bool},
+            "printer_width": {'var_type': int},
+            "write_json": {'var_type': bool},
+            "warn_error": {'var_type': str},
+            "log_format": {'var_type': str, 'accepted_values': {"text", "json", "default"}},
+            "debug": {'var_type': bool},
+            "version_check": {'var_type': bool},
+            "fail_fast": {'var_type': bool},
+            "use_experimental_parser": {'var_type': bool},
+            "static_parser": {'var_type': bool},
+        }
```

Comment:
Some of these configuration options at the moment will not affect how Cosmos will execute so it may be worth considering if we want to expose these for configuration.

For example [fail_fast ](https://docs.getdbt.com/reference/global-configs/failing-fast) for `dbt run` will prevent other models from running, however since Cosmos executes each model as its own task, it will not terminate other tasks that are running in their own subprocess. This may cause confusion for a user if one task fails on an error but the other ones are still running if they set this configuration option. The same with `use_colors` will not have an effect on task logs.

Reply:
@jbandoro Hi,
I think we can support:
- send_anonymous_usage_stats
- partial_parse
- use_experimental_parser
- static_parser
- printer_width
- write_json
- warn_error
- warn_error_options
- log_format
- debug
- version_check